# HG changeset patch
# User Test User <test@example.com>
# Date 1736311725 -39600
#      Wed Jan 08 15:48:45 2025 +1100
# Node ID 4e379f8d3278a6645a5cf706cd8555da195b089e
# Parent  0000000000000000000000000000000000000000
initial commit


add another file

diff --git a/README b/README
new file mode 100644
--- /dev/null
+++ b/README
@@ -0,0 +1,1 @@
+HELLO
diff --git a/test.txt b/test.txt
new file mode 100644
--- /dev/null
+++ b/test.txt
@@ -0,0 +1,1 @@
+TEST
